nfc/include/nfc_t4t_lib.h

```diff
@@ -42,11 +42,12 @@
  *
  * @note If you are using nRF52832 chip (in IC rev. Engineering B or
  * Engineering C) or if You are using nRF52840 chip (in IC rev. Engineering A,
- * B or C) library will use TIMER 4 to apply workarounds for the anomalies.
+ * B or C) library will use TIMER 4 to apply workarounds for the anomalies
+ * listed in @ref nfc_fixes.h
```

Comment:
I think, we no longer use this header file. Do we?

Reply:
I removed all changes in header doc. It seems like I need to also update NFC headers in the NFC repo